From 9fd62062bf36e0a9960a0d5367dd19681abe4f18 Mon Sep 17 00:00:00 2001
From: Weiwen Chen <cww@rock-chips.com>
Date: Fri, 24 Jul 2020 15:50:14 +0800
Subject: [PATCH] dma-buf: export the function set release callback

Fix if CONFIG_DRM_ROCKCHIP config module compile.

ERROR: "dma_buf_set_release_callback" [drivers/gpu/drm/rockchip/rockchipdrm.ko] undefined!
ERROR: "dma_buf_get_release_callback_data" [drivers/gpu/drm/rockchip/rockchipdrm.ko] undefined!

Signed-off-by: Weiwen Chen <cww@rock-chips.com>
Change-Id: Ic2de2b500990b6a818a976ffb87e538bb0cb9f11
---
 drivers/dma-buf/dma-buf.c | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/drivers/dma-buf/dma-buf.c b/drivers/dma-buf/dma-buf.c
index 87dd2c5b4e20..cec2f6d9c6e3 100644
--- a/drivers/dma-buf/dma-buf.c
+++ b/drivers/dma-buf/dma-buf.c
@@ -523,6 +523,7 @@ void *dma_buf_get_release_callback_data(struct dma_buf *dmabuf,
 
 	return result;
 }
+EXPORT_SYMBOL_GPL(dma_buf_get_release_callback_data);
 
 int dma_buf_set_release_callback(struct dma_buf *dmabuf,
 				 void (*callback)(void *), void *data)
@@ -544,6 +545,7 @@ int dma_buf_set_release_callback(struct dma_buf *dmabuf,
 
 	return 0;
 }
+EXPORT_SYMBOL_GPL(dma_buf_set_release_callback);
 #endif
 
 /**
-- 
2.35.3

